plugin/PlayAssetDelivery/src/test/java/com/google/play/core/godot/assetpacks/PlayAssetDeliveryTest.java

```diff
@@ -140,4 +141,41 @@ public void getAssetLocation_not_exist() {
     Dictionary resultDict = testSubject.getAssetLocation("packName", "assetPath");
     assertThat(resultDict).isEqualTo(testDict);
   }
+
+  @Test
+  public void getPackLocation_exist() {
+    PlayAssetDelivery testSubject = createPlayAssetDeliveryInstance();
+    Dictionary testDict =
+        PlayAssetDeliveryUtils.constructAssetPackLocationDictionary(
+            "~/Documents/assetsPath/", 0, "~/Documents/path/");
+
+    when(assetPackManagerMock.getPackLocation(any(String.class)))
+        .thenReturn(new AssetPackLocationFromDictionary(testDict));
+
+    Dictionary resultDict = testSubject.getPackLocation("packName");
+    assertThat(resultDict).isEqualTo(testDict);
+  }
+
+  @Test
+  public void getPackLocation_not_exist() {
+    PlayAssetDelivery testSubject = createPlayAssetDeliveryInstance();
+    Dictionary testDict = null;
+
+    when(assetPackManagerMock.getPackLocation(any(String.class))).thenReturn(null);
+
+    Dictionary resultDict = testSubject.getPackLocation("packName");
+    assertThat(resultDict).isEqualTo(testDict);
```

Comment:
Feel free to use null directly here rather than storing it temporarily in testDict.